src/components/AstronautsPage/components/AstronautsList/components/AstronautsItem/components/AstronautsItemContent/AstronautsItemContent.tsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
 import { CardContent, Typography } from '@mui/material';
-import { astronautsData } from '../../../../../../mocks/astronautsData';
+import type { AstronautsName } from '../../../../../../types/Astronauts';
 import './AstronautsItemContent.scss';
 
-export const AstronautsItemContent = () => (
+interface AstronautsItemContentProps extends AstronautsName {
+  agency: AstronautsName;
+}
```

Comment:
don't need to extend here. Just add a field to props interface

Reply:
done